dodo.py

```diff
@@ -30,7 +30,7 @@ def task_env():
     yield dict(
         name="binder",
         doc="update binder environment with docs environment",
-        file_dep=[P.DOCS_ENV, B.YARN_INTEGRITY],
+        file_dep=[P.DOCS_ENV],
```

Comment:
rather than removing, would recommend restoring `node_modules/.yarn-integrity` with `node_modules/.yarn-state.yml`... lots of ways to fail when it gets out-of-date.

Reply:
still feel like this one (and related references to it that expect "my `node_modules` right") is worth fixing.